source/Libraries/ItchioLibrary/Models/Collection.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ItchioLibrary.Models
+{
+    public class Collection
+    {
+        /// <summary>
```

Comment:
Why do some new models have comments and others don't? I don't necessarily need them, but it should be at least consistent, either have them everywhere or none at all.